### PR TITLE
RPC call : eth_coinbase Implementation #157

### PR DESCRIPTION
**Pull request by danilowhk**
- Implemented get_coinbae Rpc method

### CODE REVIEW

File: client/src/node.rs [hunk 1]
```diff
@@ -291,6 +291,13 @@ impl Node {
         Ok(self.consensus.get_header().clone())
     }
 
+    pub fn get_coinbase(&self) -> Result<CoinbaseAddress> {
```
Comment (ncitron) [inline client/src/node.rs:294]: Can we just have it return an ethers-rs `Address`? The consensus type is more just for internal use and has some special serializations. Trying to use ethers types as the outputs when possible as most people are familiar with that. Also means we won't need to export a new type.

Same goes for in `Client` and `Rpc`.

Other than that this all looks good.

Comment (danilowhk) [reply client/src/node.rs:294]: Ok!

Comment (danilowhk) [reply client/src/node.rs]: Done, please let me know if the implementation is ok

**Review by ncitron — APPROVED** [review]
LGTM!

**Pull request merged**